# Study 805 — Cokurtosis Premium 🪁📐

**Should stocks whose returns amplify the market's fat-tailed moves earn a premium?**

Fang & Lai (1997) extend the CAPM to its *fourth* moment: beyond beta (systematic variance)
and co-skewness (systematic skewness), a name's **cokurtosis with the market** — how strongly
its return co-moves with the *cube* of the market's deviation — is a priced risk. A stock that
spikes exactly when the market has a tail move offers no diversification when you need it most,
so (the theory says) it must pay a **positive** premium: **long high-cokurtosis / short
low-cokurtosis** should earn a positive spread. We take the self-contained daily version on a
liquid US cross-section (2010-01-04 → 2026-06-30, 50 names).

*Numbers below are the frozen headline (`docs/results.md`); the live cells run the fast
synthetic control. Survivorship: current-membership mega-caps — magnitudes are an upper bound.*


## 1. The idea in one picture

Cokurtosis is a *co-movement* measure, not a stand-alone tail. It asks: on the days the **market** has an extreme (fat-tailed) move, does *this* name move with it, hard? A high-cokurtosis name amplifies the market's worst days — it fails to diversify exactly when diversification matters. Four-moment CAPM says holders of that exposure should be paid for it. So sort on trailing cokurtosis with the equal-weight market; buy the high-cokurtosis names, sell the low ones.

In [1]:
import numpy as np, pandas as pd
R = dict(spread_bps=-0.15, t_nw=-0.11, hi_bps=7.39, lo_bps=7.54, gross_sharpe=-0.03)
print('long high-cokurt / short low-cokurt spread: %+.2f bps/day (NW t = %+.2f)'
      % (R['spread_bps'], R['t_nw']))
print('  high-cokurt book %+.2f bps vs low-cokurt book %+.2f bps'
      % (R['hi_bps'], R['lo_bps']))
print('  gross spread Sharpe (before cost): %.2f' % R['gross_sharpe'])

long high-cokurt / short low-cokurt spread: -0.15 bps/day (NW t = -0.11)
  high-cokurt book +7.39 bps vs low-cokurt book +7.54 bps
  gross spread Sharpe (before cost): -0.03


## 2. Is the sort even wired up? A live synthetic control

We plant the premium in a seeded toy world (`knob>0`, high cokurtosis → higher forward return) and check the detector recovers it — and that it stays *silent* on the null (`knob=0`, cokurtosis present but unpriced). No network.

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from cokurtosis import data, strategy as st
null = st.synthetic_detect(data.synthetic_panel(knob=0.0, seed=805, n_assets=40, n_days=1200))
planted = st.synthetic_detect(data.synthetic_panel(knob=0.009, seed=805, n_assets=40, n_days=1500))
print('null world   : spread NW t = %+.2f  (should be ~0)' % null['t_nw'])
print('planted world: spread NW t = %+.2f  (should light up)' % planted['t_nw'])

null world   : spread NW t = -0.67  (should be ~0)
planted world: spread NW t = +7.20  (should light up)


## 3. The honest verdict — the premium simply isn't there

On this liquid mega-cap tape the long-high-cokurt / short-low-cokurt spread is **-0.15 bps/day** with NW *t* = **-0.11** — a flat **zero**. The high-cokurtosis and low-cokurtosis books earn essentially the same return (+7.39 vs +7.54 bps/day; Welch *t* = -0.06), the observed spread sits -0.14 sd from a 1,000-permutation placebo null, and it even flips sign between eras (*t* = -1.21 / +0.61). The seeded synthetic control recovers a *planted* premium cleanly (*t* = +7.20), so this is a genuine **absence** of the Fang-Lai systematic-kurtosis premium on 50 mega-caps, not a broken sort. Higher co-moments are famously fragile out of sample; here the fourth one contributes nothing. **Signal: None**, **Tradability: Mirage**.